db/entities/attempt.go

```diff
@@ -5,22 +5,44 @@ import (
 	"encoding/json"
 )
 
+type Attempt struct {
+	ID            string            `json:"id" db:"id"`
+	EventId       string            `json:"event_id" db:"event_id"`
```

Comment:
Here, there is only EventID, but there is no distinction by EventType. Different events can be distinguished by EventType. Events are usually classified. When retrieving, EventType is also of great concern to the business.

Reply:
Sounds like a feature request for attempt paging querying. From a technical perspective, it's possible to query by event_type by joining the `events` table.  

In the roadmap, I'm trying to get rid of `EventType ` as the only routing condition. As a gateway, routing should not only limited by the EventType.